README.md

```diff
@@ -1 +1,58 @@
-# fsleyes-plugin-shimming-toolbox
\ No newline at end of file
+# fsleyes-plugin-shimming-toolbox
+
+## Installation
+
+In the `fsleyes-plugin-shimming-toolbox` folder, run:
+
+```
+make install
+```
+
+## Running
+
+In the `fsleyes-plugin-shimming-toolbox` folder, run:
+
+```
+make run
+```
+
+
+### Developers
+
+
+First, you will need to create a `conda` virtual environment, and activate it.
+
+```
+conda create --name pst_venv
```

Comment:
fpst_venv?

Reply:
**f**sleyes-**p**lugin-**s**himming-**t**oolbox...but I only changed it cause I didn't want to overwrite my environment aha. We should sort out the environment names.

Reply:
I might not have been clear sorry, I meant it's missing the "f" in the doc

Reply:
Fixed here: https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox/pull/2/commits/9c0354f190cd7081c8077c8427d9c90457d50bcf (see above)